tendrl/monitoring_integration/sync/gluster_cluster_details.py

```diff
@@ -41,43 +33,33 @@ def get_cluster_details(integration_id):
     return cluster_details, constants.GLUSTER_DASHBOARDS
 
 
-def get_node_details(cluster_key):
+def get_node_details(integration_id):
     node_details = []
-    try:
-        node_list = utils.get_resource_keys(cluster_key, "nodes")
-        for node_id in node_list:
```

Comment:
@shtripat don't remove try catch here if suppose anyone resource gives exception then dashboard for another type resource won't be created. Each detail should be independent. Because Cluster can be imported without volume and brick details also. Lot of places ttl are used so one details not present like volume not present the dashboard creation won't happen for other.

The event we write safe code better we keep try catch. Because resource collection is a costly operation. so better we dashbaord for successfully collected resources.

Reply:
For example, volume details updated by gluster integration but if sub volume details updation takes time then it will give EtcdKeyNotFoundError in get_volume_details function. Then dashboard won't created for others. Like this i faced lot of issues.

Reply:
We are not using individual keys to read details now. Now `load()` and `load_all()` functions are used and they would return None and empty list respectively if key not found. If these functions don't work properly its a fix for these functions and not the reading code to handle `EtcdKeyNotFound` etc.

I accept that for `etcd_utils.read()` I would need to add try-except for `EtcdKeyNotFound`. Will do the same.